main/src/library/Map.flix

```diff
@@ -443,7 +378,7 @@ namespace Map {
     ///
     /// Returns the difference of `m1` and `m2`, i.e. `m1 - m2`.
     ///
-    /// That is, returns the map `m1` with the keys removed that are in `m2`.
+    /// That is, returns the Map `m1` with the keys removed that are in `m2`.
```

Comment:
I guess there was an unexpected search replace for Map that changed all these comments.